internal/hardware/retrieve.go

```diff
@@ -46,7 +27,8 @@ func (c *Client) Retrieve() *cobra.Command {
 
 			header := []string{"ID", "Facility", "Metro", "Plan", "Created"}
 
-			inc := []string{}
+			inc := c.Servicer.Includes(nil)
```

Comment:
This should be moved to after the `switch` block a couple lines down so that we keep the user-specified includes in addition to the ones that are set automatically by the CLI.